src/test/java/org/jpeek/skeleton/SkeletonTest.java

```diff
@@ -61,6 +62,25 @@ public void createsXml() {
         ).affirm();
     }
 
+    @Test
+    @Disabled
+    public void skeletonShouldReflectExactOverloadedCalledMethod() {
+        new Assertion<>(
+            "Must overload bar's methods",
```

Comment:
@dgarus I don't think that this message is clear enough; what are we trying to assert here?

Reply:
changed the message